service.libraryautoupdate/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.pyo
```

Comment:
please drop this file

Reply:
Will do. I had manually removed it in all the other ones but missed it on the last update. This is like commit number 10 so just missed it on this last one.